CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+- Adds `functions.logger` SDK to enabled structured logging in the Node.js 10 runtime. Example:
+
```

Comment:
A nit, but in a text flow we prefer "For example:"

Reply:
Done.